tests/legacy/system/test_edb_configuration_2p0.py

```diff
@@ -385,10 +385,28 @@ def test_05f_ports_between_two_points(self, edb_examples):
         edbapp.close()
 
     def test_06_s_parameters(self, edb_examples):
-        with open(self.local_input_folder / "s_parameter.json") as f:
-            data = json.load(f)
-        data["general"]["s_parameter_library"] = self.local_input_folder
-
+        data = {
+            "general": {"s_parameter_library": self.local_input_folder},
+            "s_parameters": [
+                {
+                    "name": "GRM32_DC0V_25degC_series",
+                    "file_path": "GRM32_DC0V_25degC_series.s2p",
+                    "component_definition": "CAPC3216X180X55ML20T25",
+                    "apply_to_all": True,
+                    "components": [],
+                    "reference_net": "GND",
+                },
+                {
+                    "name": "GRM32_DC0V_25degC_series",
+                    "file_path": "GRM32_DC0V_25degC_series.s2p",
+                    "apply_to_all": False,
+                    "component_definition": "CAPC3216X190X55ML30T25",
+                    "components": ["C59"],
+                    "reference_net": "GND",
+                    "reference_net_per_component": {"C59": "GND"},
+                },
+            ],
+        }
```

Comment:
Why are you defining data manually instead of using `s_parameter.json` like before ?

Reply:
This is for the sake of debugging. If something goes wrong, I can check everything in one place instead of jumping between different files.